sklearn/preprocessing/data.py

```diff
@@ -1538,10 +1540,45 @@ def transform(self, X):
                         columns.append(bias)
                 XP = sparse.hstack(columns, dtype=X.dtype).tocsc()
             else:
+                # When the matrix is small, it is faster to transpose
+                # the matrix first to multiply contiguous memory segments.
+                n = X.shape[1]
```

Comment:
Please rename to `n_features` to make the loop iteration below more explicit.

Reply:
Done